packages/cdk8s-plus-17/test/job.test.ts

```diff
@@ -12,6 +12,33 @@ test('defaultChild', () => {
 
 });
 
+test('Allows setting all options', () => {
+
+  const chart = Testing.chart();
+
+  const job = new kplus.Job(chart, 'Job', {
+    containers: [new kplus.Container({ image: 'image' })],
```

Comment:
```suggestion
    containers: [{ image: 'image' }],
```

https://github.com/awslabs/cdk8s/pull/404 :)